packages/s2-core/src/sheet-type/spread-sheet.ts

```diff
@@ -381,6 +381,11 @@ export abstract class SpreadSheet extends EE {
     } else {
       this.options = customMerge(this.options, options);
     }
+
+    if (reset || options.tooltip?.renderTooltip) {
```

Comment:
这种情况, 是不是 init 之前把上一次的 tooltip 给 销毁掉要好一点?

```suggestion
  private initTooltip() {
    this.tooltip?.destroy?.()
    this.tooltip = this.renderTooltip();
    ...
  }
```